### PR TITLE
Backport 2.6.10 into master

### PR DESCRIPTION
**Pull request by Kdecherf**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | 
| Documentation | no
| Translation   | no
| CHANGELOG.md  | no
| License       | MIT

### CODE REVIEW

File: web/wallassets/41a1d465797f51702171.otf
Comment (yguedidi) [inline web/wallassets/41a1d465797f51702171.otf:1]: Not sure this should be commited

**Review by yguedidi — CHANGES_REQUESTED** [review]

File: web/wallassets/41a1d465797f51702171.otf
Comment (Kdecherf) [reply web/wallassets/41a1d465797f51702171.otf:1]: It comes from bd8ccf924f4ae78be407b151b668679edd511e0b #7754 
@nicosomb is this an error?

Comment (j0k3r) [reply web/wallassets/41a1d465797f51702171.otf:1]: Hum, assets should not be in the repo anymore.
What if you remove it and then generated assets, does that file exist?

Comment (Kdecherf) [reply web/wallassets/41a1d465797f51702171.otf:1]: As it was a PITA to redo the merge commit, I went with a separate commit to remove it

**Comment by Kdecherf** [comment]
I'm not a big fan of what I've done but here we go, I've "rebased" my PR to match the current state of `master` with a merge commit. I think it is the quickest option instead of redoing merge conflict resolution.

File: src/Command/Import/ImportCommand.php [hunk 1]
```diff
@@ -61,7 +63,8 @@ public function __construct(
         WallabagV1Import $wallabagV1Import,
         ElcuratorImport $elcuratorImport,
         ShaarliImport $shaarliImport,
-        PocketHtmlImport $pocketHtmlImport
+        PocketHtmlImport $pocketHtmlImport,
+        OmnivoreImport $omnivoreImport
```
Comment (yguedidi) [inline src/Command/Import/ImportCommand.php:67]: nitpick: would have added it after DeliciousImport, like the property and initialization in constructor, and like it's done in 2.6 😅

**Review by yguedidi — APPROVED** [review]
what a work! congrats for this great work!

**Pull request merged**